src/Tempest/Http/Session/Managers/FileSessionManager.php

```diff
@@ -82,7 +84,15 @@ private function resolve(SessionId $id): ?Session
         try {
             $content = @file_get_contents($path);
 
-            return unserialize($content);
+            return unserialize($content, ['allowed_classes' => [
+                SessionId::class,
```

Comment:
these are the classes the tests revealed. but we probably need to add every rule to this?

I did not go as far (yet) as this is what I initially found

Reply:
Hmm, technically it could be anything. I don't think we can add filtering here.